datasets/humor-patterns.yaml

```diff
@@ -0,0 +1,12 @@
+Name: Humor patterns
+Description: Patterns used for quering Alexa traffic when creating the taxonomy described in the paper "“Alexa, Do You Want to Build a Snowman?” Characterizing Playful Requests to Conversational Agents"  by C. Shani, A. Libov, S. Tolmach, L. Lewin-Eytan, Y. Maarek, and D. Shahaf (CHI LBW 2022)
+Contact: alibov@amazon.com ########## Alex?
+Tags:

```

Comment:
@chenxshani please use approved tags from the list at https://github.com/awslabs/open-data-registry/blob/main/tags.yaml